app/helpers/container_helper/textual_summary.rb

```diff
@@ -32,4 +32,8 @@ def textual_restart_count
   def textual_backing_ref
     {:label => "Backing Ref (Container ID)", :value => @record.backing_ref}
   end
+
+  def textual_command
+    {:label => "Command", :value => @record.container_definition.command  } if @record.container_definition.command
```

Comment:
@moolitayer it's nice to show the command in a shell-like output:

    require 'shellwords'

    ...
    :value =>  Shellwords.join(JSON.parse(@record.container_definition.command))